src/app/components/menubar/menubar.ts

```diff
@@ -1050,10 +1050,6 @@ export class Menubar implements AfterContentInit, OnDestroy, OnInit {
 
             anchorElement ? anchorElement.click() : element && element.click();
 
-            const processedItem = this.visibleItems[this.focusedItemInfo().index];
```

Comment:
mmm I'm not sure if this is ok for accessibility. 
@TaneliTuomola could you check it, please? :)

Reply:
I am aware there is a known issue(#15048) for accessibility for keyboard navigation whilst using Microsoft narrator. That issue I believe still needs to be resolved.